src/renderer.cpp

```diff
@@ -556,11 +596,36 @@ void Renderer::init()
         true
     );
 
+    backendProgramHandles[RendererProgram::LIGHTING_FLAT] = bgfx::createProgram(
+        getShader(vertexLightingPathFlat.c_str()),
+        getShader(fragmentLightingPathFlat.c_str()),
+        true
+    );
+
+    backendProgramHandles[RendererProgram::LIGHTING_SMOOTH] = bgfx::createProgram(
+        getShader(vertexLightingPathSmooth.c_str()),
+        getShader(fragmentLightingPathSmooth.c_str()),
+        true
+    );
+
+    backendProgramHandles[RendererProgram::FIELD_SHADOW] = bgfx::createProgram(
+        getShader(vertexFieldShadowPath.c_str()),
+        getShader(fragmentFieldShadowPath.c_str()),
+        true
+    );
+
+    shadowMapProgramHandle = bgfx::createProgram(

```

Comment:
Can you please add this to the `backendProgramHandles` too? Thanks!

Reply:
Done!